### ISSUE TITLE
[FO] Quantity touchspin can break and buttons are too small

### ISSUE THREAD

**Issue by SharakPL** [PrestaShop/PrestaShop#26641]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

1. Quantity touchspin can brake on product page, product quick view on the list and cart summary.
2. More/less buttons are definitely too small, especially on mobile. Recommended size for any button to be usable on mobile is 40+ pixels. These ones have only 23-24 px, so using them with a finger is quite a challange.

![obraz](https://user-images.githubusercontent.com/5007456/142082604-827d6274-de1a-4f3a-ba1c-9f0a77704ec1.png)

![obraz](https://user-images.githubusercontent.com/5007456/142083246-534de117-4a30-4882-9775-686df66c6126.png)

### Expected behavior

Quantity component with buttons should be consistent. Buttons should be bigger, especially on mobile.

### Steps to reproduce

1. Open product page, product quick view in category/home page or cart checkout
2. Open browser dev tools, enable responsive mode and check viewport sizes when it breaks

### PrestaShop version(s) where the bug happened

1.7.x.x

**Comment by hibatallahAouadni** [comment]
Hello @SharakPL 

On iPhone 11 seems good to me, see the attached screenshot below: 

![image](https://user-images.githubusercontent.com/23308427/142298654-8972809a-695b-463a-b43b-7911ed611b5a.jpeg)

Could you please provide the size of the device that you are testing on it 🙏 
Please check and feedback.

Thanks!

**Comment by SharakPL** [comment]
On english version it breaks below 351px on product page and 426px on cart page, but there are variables changing these numbers, eg. translations. Here's polish version of product page, where it breaks:

below 405px:

![obraz](https://user-images.githubusercontent.com/5007456/142301808-6054614f-374c-4f75-b5b1-b77bc9680707.png)


and even on tablets (768-991px):

![obraz](https://user-images.githubusercontent.com/5007456/142301957-31e8fad2-051e-4916-af3a-f8050b96301b.png)

My point is it shouldn't break at all, no matter the size or language, or your iPhone 😛

**Comment by khouloudbelguith** [comment]
Hi @SharakPL,

I manage to reproduce the issue with PS1769, PS1778 and develop branch.
I have the issue on the Product page 
![image](https://user-images.githubusercontent.com/16067358/142427491-9a4cb32f-2efa-4174-b24f-4b74298535b4.png)
I have the issue on the Shopping cart page
![image](https://user-images.githubusercontent.com/16067358/142427571-ddbe091a-deed-48be-bcf4-f1754cf4a019.png)

I'll add this to the debug backlog so that it's fixed.

Please be aware that due to the high number of reported issues, some bugs might take a very long time to be resolved. If this issue is important to you and you cannot wait for it to be fixed on the project's own time, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you. 

If you manage to get the problem fixed, then please contribute the fix back to the project as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/). Remember that the more people contribute, the better PrestaShop becomes for everyone.

Thanks!

**Comment by matks** [comment]
Hi @SharakPL is this issue valid on Hummingbird ? 😉

**Comment by SharakPL** [comment]
I don't think so. Bootstrap-touchspin isn't used anymore (at least [it shouldn't](https://github.com/PrestaShop/hummingbird/blob/4f112bebad1825d40e0578af409c786e7da39e96/src/scss/theme.scss#L7C72-L7C72) 🤔) and quantity input group doesn't brake.

**Comment by hibatallahAouadni** [comment]
> Hi @SharakPL is this issue valid on Hummingbird ? 😉

The bug doesn't occur on HummingBird Theme.
